lib/rules/aria-input-field-label.json

```diff
@@ -0,0 +1,18 @@
+{
+	"id": "aria-input-field-label",
+	"selector": "[role=\"combobox\"], [role=\"listbox\"], [role=\"searchbox\"], [role=\"slider\"], [role=\"spinbutton\"], [role=\"textbox\"]",
+	"matches": "aria-form-field-label-matches.js",
+	"tags": ["wcag2a", "wcag412"],
+	"metadata": {
+		"description": "Ensures every ARIA input field has an accessible name",
+		"help": "ARIA input fields have an accessible name"
+	},
+	"all": [],
+	"any": [
+		"aria-label",
+		"aria-labelledby",
+		"non-empty-title",
+		"has-visible-text"
```

Comment:
This doesn't belong here. These elements don't have name from content. Because this check is a little off, I don't actually think you have to change any of the test cases.